_source/pages/help/Help.scss

```diff
@@ -0,0 +1,14 @@
+.help-intro {
+  margin-bottom: 32px;
+}
+
+.help-container {
```

Comment:
`help__container`

Reply:
container is not an element (block__element--modifier) but a block element itself as it contains other elements. Or do you use an own naming system?